zoomus/components/report.py

```diff
@@ -41,6 +41,15 @@ def get_user_report(self, **kwargs):
             "/report/users/{}/meetings".format(kwargs.get("user_id")), params=kwargs
         )
 
+    def get_meeting_detail_report(self, **kwargs):
```

Comment:
Thank you for this addition! Can you please add a test case for it so that we can ensure that we are maintaining test coverage?

Thank you for your time and effort!